packages/fxa-auth-server/test/oauth/routes/token.js

```diff
@@ -74,25 +73,23 @@ describe('/token POST', function () {
         },
         (err) => {
           assert.equal(err, null);
-          done();
```

Comment:
Was there a reason these done() calls were removed. If this is now necessary, I think the call to v shold be awaited since it's async.

Reply:
Ah ok, ironically, adding `done` back in causes the tests to fail with: ` Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.`.